### PR TITLE
feat: added the committer info to the commit

### LINKED ISSUE #2676: Wrong user mentioned in git commit when content is deleted
When users (with git-gateway and netlify identify) delete content, the commit message says that I deleted the content, first I thought I was hacked or something, it's scary seeing your account doing something while you're asleep : )

In the future I would like to know who removed something...

### PR DESCRIPTION
**Pull request by KancerEzeroglu**
closes #2676 

**Summary**
When users (with git-gateway and netlify identify) delete content, the commit message says that I (github account owner) deleted the content.
We would like to know who made change something.

I used the gatsby starter netlify cms project to repeat the problem. I created, deleted, and edited a blog with another user. For each action, I saw my own name in my Github account, instead of seeing the user's name.
After making my local Netlify cms project usable connected with GitHub, I added the committer info into the commit.
Then I repeated the same manual tests. 
When I checked the commits, saw the user name that made a change. 
The change I made worked!

**Test plan**
for now, I don't have

**A picture of a cute animal (not mandatory but encouraged)**

giphy

### CODE REVIEW

**Comment by KancerEzeroglu** [comment]
@erezrokah , I've checked the code but couldn't find where is the problem that makes tests failed.
Do you have an idea?

File: packages/netlify-cms-backend-git-gateway/src/GitHubAPI.ts [hunk 1]
```diff
@@ -91,12 +94,20 @@ export default class API extends GithubAPI {
...
       author?: { name: string; date: string };
+      committer?: { name: string; date: string };
     } = {
       message,
       tree: changeTree.sha,
       parents: changeTree.parentSha ? [changeTree.parentSha] : [],
     };
 
+    if (this.committer) {
+      commitParams.committer = {
```
Comment (erezrokah) [inline packages/netlify-cms-backend-git-gateway/src/GitHubAPI.ts:105]: `committer` defaults to `author`, see https://developer.github.com/v3/git/commits/#create-a-commit.

So this seems redundant

**Review by erezrokah — COMMENTED** [review]
I think #2676 was fixed by switching to deleting files using the commits API (instead of delete file API).
The change was done as a part of https://github.com/netlify/netlify-cms/pull/4139

**Comment by erezrokah** [comment]
Closing the PR and the issue, thanks @KancerEzeroglu and sorry for the late review

**Pull request closed without merging**